src/indev/lv_indev.c

```diff
@@ -360,6 +360,11 @@ void lv_indev_set_display(lv_indev_t * indev, lv_display_t * disp)
     indev->disp = disp;
 }
 
+void lv_indev_set_long_press_time(lv_indev_t * indev, uint16_t long_press_time)
+{
```

Comment:
Other APIs have check of NULL pointer. We may add it here too?
```
    if(indev == NULL) return;
```

Reply:
done !